user_sync/rules.py

```diff
@@ -780,14 +812,14 @@ def read_remove_list(file_path, delimiter = None, logger = None):
             id_type = row.get(id_type_column_name)
             user = row.get(user_column_name)
             domain = row.get(domain_column_name)
-            user_key = RuleProcessor.get_user_key(user, domain, id_type)
+            user_key = RuleProcessor.get_user_key(user, domain, None, id_type)
```

Comment:
I thought it didn't, but in the current v1 it does seem to (it now wants an email input as well?). It was crashing in read_remove_list, because it complained it only provided 3 arguments but expected 4.